### ISSUE TITLE
consolidate Color() calls for spectator

### ISSUE THREAD

**Issue by EntranceJew** [TTT-2/TTT2#1481]
a lot of Color() calls reinvent color values for things which already exist or are so isolated that they can't be reached and shared, so they should be consolidated

**Comment by Histalek** [comment]
We might want to look at our colors in general:

```shell
$ rg ' Color\(' --stats --quiet

192 matches
192 matched lines
68 files contained matches
972 files searched
```

We have 192 calls to `Color()` in 68 different files in our codebase. at a glance i can even spot a few duplicates.
Consolidating and moving the creation of these colors to a single place might be a good idea. Even if it is only to give them meaningful names.

**Comment by EntranceJew** [comment]
agreed, my single biggest pain point with extending the UI was being able to reach the color variations from the VSKIN -- otherwise i could attach all the parts, i just couldn't style it to part

**Comment by saibotk** [comment]
I think this needs more investigation than just the spec color :)

**Comment by TimGoll** [comment]
I renamed the issue to reflect that
